src/test/store/symbolication.test.js

```diff
@@ -0,0 +1,187 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
I broke this out into a separate file, as it had a bit of cross-cutting concerns.